tests/data/string_prefixes.py

```diff
@@ -1,18 +1,24 @@
-#!/usr/bin/env python3.6
+#!/usr/bin/env python3
 
-name = R"Łukasz"
```

Comment:
Any reason to change the input section here?

Reply:
Just for reducing a redundant test, since all the prefixes are tested below! I thought about removing it altogether, but seems fine to test f-string substitution there at least. Would you like to keep it?